org.scala-ide.sdt.debug.expression/src/org/scalaide/debug/internal/expression/proxies/JdiProxy.scala

```diff
@@ -95,10 +93,10 @@ trait JdiProxy extends Dynamic {
 /**
  * Base for companion objects for [[org.scalaide.debug.internal.expression.proxies.JdiProxy]].
  */
-trait JdiProxyCompanion[Proxy <: JdiProxy, Underlying <: Value] {
+trait JdiProxyCompanion[P <: JdiProxy, Underlying <: Value] {
```

Comment:
Why rename `Proxy` to `P`? I see no warning in this file, and those names were descriptive for a reason.